ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDDefendant.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.cmc.ccd.domain;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class CCDDefendant implements CCDPartyElement {
+    private CCDPartyType claimantProvidedType;
+    private String claimantProvidedEmail;
+    private CCDAddress claimantProvidedServiceAddress;
+    private String claimantProvidedName;
+    private String claimantProvidedPhoneNumber;
+    private CCDAddress claimantProvidedAddress;
+    private CCDAddress claimantProvidedCorrespondenceAddress;
+    private String claimantProvidedDateOfBirth;
+    private String claimantProvidedContactPerson;
+    private String claimantProvidedCompaniesHouseNumber;
+    private String claimantProvidedTitle;
+    private String claimantProvidedBusinessName;
```

Comment:
I don't see any of the above `claimantProvided...` party details being mapped anywhere.